js/adapt-contrib-glossary.js

```diff
@@ -21,10 +21,10 @@ define([
      * <a href='#' data-glossaryterm='term name'>glossary term link</a>
      * these links should trigger the glossary to open with that term automatically selected
      */
-    $('body').on('click.glossary', 'a[data-glossaryterm]', function(e) {
+    $('body').on('click.glossary', 'a[data-glossaryterm]', (e) => {
       if (e) e.preventDefault();
 
-      var newoptions = _.clone(options);
+      const newoptions = _.clone(options);
```

Comment:
I see the framework dependency has since been bumped; FYI you can now do something like:
```js
const newoptions = {
  ...options,
  attributes: {
    'data-termtoshow': e.currentTarget.getAttribute('data-glossaryterm')
  }
};
```